app/models/installed_gas_asset.rb

```diff
@@ -10,6 +11,10 @@ class InstalledGasAsset
   attribute :lifetime, Integer
   attribute :investment_cost, Float
 
+  alias_method :initial_investment, :investment_cost
+  alias_method :technical_lifetime, :lifetime
+  alias_method :units, :amount
```

Comment:
Maybe it would be worthwhile to rename these attributes to match those in Technology?